### ISSUE TITLE
How to promote a deployment to the main domain?

### ISSUE THREAD

**Issue by nandubatchu** [milliHQ/terraform-aws-next-js#343]
Tried to set the alias - but not fruitful.
```
tf-next alias set mydomain.com <deployment-id>.mydomain.com
```

**Comment by lesnitsky** [comment]
#341

**Comment by nandubatchu** [comment]
@lesnitsky I am not referring to the subdomain. Subdomains are working fine. I am asking about the main domain!

**Comment by lesnitsky** [comment]
the issue is still valid for the subdomain, I think the root cause is the same

**Comment by nandubatchu** [comment]
Thanks @lesnitsky - adding the CloudFront invalidation rule solved my issue! 😄 

But this change needs to be picked up for implementation!

**Comment by ofhouse** [comment]
Yes the invalidation part needs a bigger refactoring as we need to reimplement the existing system for working with the new atomic deployments.
